commit 8a90ff450b11dcc6dc855040805093e1ef432092
Author: boombatower <jimmy@boombatower.com>
Date:   Thu Mar 8 18:27:23 2018 -0600

    Make rpm-lint happy by not using env as interpreter.

diff --git a/git-sync b/git-sync
index 4889331..cefe4d0 100755
--- a/git-sync
+++ b/git-sync
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
 #
 # git-sync
 #
